src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -231,6 +244,12 @@ class JoomlacmsPullsListener extends AbstractListener
 			'^administrator/components/com_fields',
 			'^components/com_fields',
 		],
+		// Composer Change
+		'72' => [
```

Comment:
Done.
Why it helps to improve? Well, numbers are easily miss-typed without any chance to notice (they still are correct numbers). With constants, that can not happen, and you get code completion for the possible values. You can understand their meaning immediately without having to consult a list with the description of numbers.